unit/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -1241,3 +1241,35 @@ TEST_CASE(
     CHECK(converted == expected);
   }
 }
+
+TEST_CASE("pointer_offset_exprt to SMT conversion", "[core][smt2_incremental]")
+{
+  const symbol_exprt foo{"foo", pointer_typet(unsigned_int_type(), 64)};
+
+  SECTION("simple pointer_offset_exprt conversion")
+  {
+    const pointer_offset_exprt pointer_offset_exp =
+      *expr_try_dynamic_cast<pointer_offset_exprt>(pointer_offset(foo));
```

Comment:
See previous comments on `*expr_try_dynamic_cast` and `pointer_offset` in the other unit test.